pkg/models/applications/applications.go

```diff
@@ -46,7 +46,7 @@ type Application struct {
 	VersionId   string            `json:"version_id" description:"application version id"`
 	Status      string            `json:"status" description:"application status"`
 	UpdateTime  time.Time         `json:"updateTime" description:"update time"`
-	CreateTime  time.Time         `json:"createTime" description:"create name"`
+	CreateTime  time.Time         `json:"createTime" description:"create time"`
```

Comment:
Should it be UpdatedTime, CreatedTime?

Reply:
the field is also used in openpitrix, should be different?